reference/Upload-API.yaml

```diff
@@ -1448,5 +1433,138 @@ paths:
               type: object
               properties: {}
       description: 'Returns the list of entitlement and Time to live (TTL) for the "entry" identify by the entryid. '
+  '/api/v1/{apiUserId}/{apiKey}/{languageCode}/feed/{feedId}/':
```

Comment:
Can you add some description to this spec please? Explaining in brief what does this API do.